### ISSUE TITLE
should the brata for ios be a peer level repository?

### ISSUE THREAD

**Issue by mmieczkowski** [brata-hsdc/brata#99]
I need to import the brata for iOS repository to brata-hsdc. I can clone the brata repository and add the ios code it to the path brata-hsdc/brata/BRATA_iOS along side the android repository. However, the android project is probably unwanted by iOS developers and vica-versa.   I'm thinking this should be a peer level repository parallel to brata-hsdc/brata rather than a part of it.

i.e.
- brata-hsdc
  - /brata
    - /BRATA
    - README.md
  - /brata_ios
    - /BRATA
    - README.md

thoughts?

**Comment by ellerychan** [comment]
Makes sense to me to put it in a separate **brata_ios** repo, and eventually rename **brata** to **brata_android**.  But I will defer to @DragonmasterJ and @jawaad-ahmad for those types of decisions.

**Comment by jawaad-ahmad** [comment]
Is there any chance of the two sharing any code, or perhaps using
cross-development tools such as Sencha, Xamarin, or PhoneGap?

If we do use those, we'll be passing along their complexities to the
students, which I think we don't want to do, so I imagine we will continue
to support two separate baselines.

There are those who would argue why we have three separate repositories in
the first place; our code base isn't expected to grow as fast as or to the
same size as some larger projects, so we could have kept everything in the
same repository anyway.

It makes no difference to me personally where the code lives on this one;
both ways make equal sense.

Let's think about how we want to organize this. Do the iOS issues need to
be kept in a separate tracker? Does it need its own dedicated wiki or
GitHub pages? Will the wiki contain nearly identical pages for both
implementations?

When we change our interface and need to modify the Brata side, do we want
to open one issue and tag to two labels, or open two identical issues in
two separate trackers?

**Comment by mmieczkowski** [comment]
I don't think cross development tools are in the best interest of the students. Not at the high school level anyway. a separate tracker sounds like a good idea but I don't think we need a separate wiki or github pages.

**Comment by jawaad-ahmad** [comment]
It all comes in one package--repo, tracker, wiki, pages.

Even if there were shared issues, they would be applied in multiple places
so it makes sense to have individual issues. The individual issues could
live in separate trackers or in the same tracker with separate labels.

Both ways seem equivalent to me.

If there is a strong preference going one way, then let's do that.

**Comment by mmieczkowski** [comment]
an iOS label in the tracker is find with me. Can we apply an Android label to the open existing brata android issues so that developers can find the issues for the platform they are working on?

**Comment by jawaad-ahmad** [comment]
We currently have the `brata-framework` label; I created a `brata-ios` label as well. Any recommendations to rename these two? Maybe rename `brata-framework` to `brata-android`?

I went and marked all the ones I think apply to `brata-framework`, `brata-ios`, or both. If you're porting the current Android BRATA to iOS, then you'll need to be aware of the current limitations during the port. Please feel free to remove the label from issues for which it doesn't apply.

**Comment by mmieczkowski** [comment]
Great. I'll upload the source along side the android repo. I don't think I can just push my local git repo, I think I have to transpose the head revision and files to the android repo, correct?
